clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/validators/securitygroup/UpsertOpenstackSecurityGroupDescriptionValidator.groovy

```diff
@@ -33,19 +34,37 @@ import org.springframework.validation.Errors
 @Component
 class UpsertOpenstackSecurityGroupDescriptionValidator extends AbstractOpenstackDescriptionValidator<UpsertOpenstackSecurityGroupDescription> {
 
+  static final int MIN_PORT = -1
+  static final int MAX_PORT = (1 << 16) - 1
```

Comment:
I had moved it out because while max port is the same, min port isn't the same. A load balancer wants a valid min port (1), a security group can have a min port of -1, meaning all ports. So I kept the reverent max next to the different min ports.